sdk/harambe/types.py

```diff
@@ -17,6 +18,7 @@
 AsyncScraperType = Callable[["SDK", URL, Context], Awaitable[None]]  # type: ignore # noqa: F821
 SetupType = Callable[["SDK"], Awaitable[None]]  # type: ignore # noqa: F821
 Callback = Callable[..., Awaitable[None]]
+PageCallback = Callable[[Page], Awaitable[None]]
```

Comment:
```suggestion
PageCallback = Callable[[AbstractPage], Awaitable[None]]
```

so it works with soup